docs/content/how-to/debug-nodes.md

```diff
@@ -45,13 +45,13 @@ you can check the following:
    each machine. You can do this by first creating a bastion machine that will allow you to ssh to worker nodes
    and running a utility script that will download logs from the machines.
 
-    Extract the public/private key for the cluster. If you created the cluster with the --generate-ssh flag, a
+    Extract the public/private key for the cluster. If you created the cluster with the --generate-ssh flag, an
```

Comment:
presumably it's fine as it is
https://www.a-or-an.com/a_an/ssh

Reply:
Ok I reverted that back.